cypress/features/profile.feature

```diff
@@ -8,6 +8,113 @@ Feature: Profile
+    | activity     | 08:10  | 11:25  | activity-1  | location-1 |
+    | activity     | 11:25  | 12:00  | activity-2  | location-2 |
+    | activity     | 13:35  | 16:45  | activity-3  | location-3 |
+    And I log into the hub
+    Then I am shown the following card
+    | dataTest  | morningEvents       |
+    | title     | Morning             |
+    | content   | activity-1          |
+    | content   | 8:10am to 11:25am   |
+    | content   | Location-1          |
+    | content   | activity-2          |
+    | content   | 11:25am to 12:00pm  |
+    | content   | Location-2          |
+    Then I am shown the following card
+    | dataTest  | afternoonEvents    |
+    | title     | Afternoon          |
+    | content   | activity-3         |
+    | content   | 1:35pm to 4:45pm   |
+    | content   | ocation-3          |
+    Then I am shown the following card
+    | dataTest  | eveningEvents           |
+    | title     | Evening                 |
+    | content   | No activities scheduled |
+  
+  Scenario: I want to view my incentives
+    Given that I go to the "profile" page
+    When I have the following incentives
+    | iepLevel | iepDate     |
+    | Basic    | 2017-03-08  |
+    And I log into the hub
+    Then I am shown the following card
+    | dataTest  | currentLevel         |
+    | title     | Your current incentive level is  |
+    | content   | Basic |
+    Then I am shown the following card
+    | dataTest  | reviewDate                |
+    | title     | This can be reviewed from |
+    | content   | Thursday 8 June 2017      |
+    Then I am shown the following card
+    | dataTest  | incentivesLink                    |
+    | title     | Read more about incentive levels  |
+
+  Scenario: I want to view a summary of my money
+    Given that I go to the "profile" page
+    When I have the following money summary
+    | spends    | 123.45  |
+    | cash      | 5       |
+    | savings   | 9       |
+    | damageObligations | 0 |
+    | currency  | GBP   |
+    And I log into the hub
+    Then I am shown the following closed sensitive card
+    | dataTest  | moneySpends                     |
+    | title     | Spends                          |
+    | content   | Spends account current balance  |
+    | sensitive | £123.45                         |
+    Then the "moneySpends" sensitive card is closed
+    When I click on the "moneySpends" card
+    Then the "moneySpends" sensitive card is open
+    Then I am shown the following closed sensitive card
+    | dataTest  | moneyPrivate                    |
+    | title     | Private                         |
+    | content   | Private account current balance |
+    | sensitive | £5.00                           |
+    Then the "moneyPrivate" sensitive card is closed
+    When I click on the "moneyPrivate" card
+    Then the "moneyPrivate" sensitive card is open
+    Then I am shown the following closed sensitive card
+    | dataTest  | moneySavings                    |
```

Comment:
meh.. the levels of abstraction in these tests seem quite varied.

could call this attribute `card | moneySavings` or something? 

`dataTest` seems to be very specific, and referring to this as a card would raise the level of abstraction out of